src/app/ReadClient.h

```diff
@@ -144,30 +144,31 @@ class ReadClient : public Messaging::ExchangeDelegate
      *  The ReadClient will automatically shut itself down when it receives a
      *  response or the response times out.  So manual shutdown is only needed
      *  to shut down a ReadClient before one of those two things has happened,
-     *  (e.g. if SendReadRequest returned failure).
+     *  (e.g. if SendReadRequestMessage returned failure).
      */
     void Shutdown();
 
     /**
      *  Send a Read Request.  There can be one Read Request outstanding on a given ReadClient.
-     *  If SendReadRequest returns success, no more Read Requests can be sent on this ReadClient
+     *  If SendReadRequestMessage returns success, no more Read Requests can be sent on this ReadClient
      *  until the corresponding InteractionModelDelegate::ReadDone call happens with guarantee.
      *
      *  @retval #others fail to send read request
      *  @retval #CHIP_NO_ERROR On success.
      */
-    CHIP_ERROR SendReadRequest(ReadPrepareParams & aReadPrepareParams);
+    CHIP_ERROR SendReadRequestMessage(ReadPrepareParams & aReadPrepareParams);
 
     /**
      *  Send a subscribe Request.  There can be one Subscribe Request outstanding on a given ReadClient.
-     *  If SendSubscribeRequest returns success, no more subscribe Requests can be sent on this ReadClient
+     *  If SendSubscribeRequestMessage returns success, no more subscribe Requests can be sent on this ReadClient
      *  until the corresponding InteractionModelDelegate::ReadDone call happens with guarantee.
      *
      *  @retval #others fail to send subscribe request
      *  @retval #CHIP_NO_ERROR On success.
      */
-    CHIP_ERROR SendSubscribeRequest(ReadPrepareParams & aSubscribePrepareParams);
-    CHIP_ERROR OnUnsolicitedReportData(Messaging::ExchangeContext * apExchangeContext, System::PacketBufferHandle && aPayload);
+    CHIP_ERROR SendSubscribeRequestMessage(ReadPrepareParams & aSubscribePrepareParams);
```

Comment:
Again, not sure this should have the "Message" suffix.

Reply:
sure, rename it back.